server/frontend/src/components/ActivityGraph.vue

```diff
@@ -0,0 +1,119 @@
+<template><span></span></template>
+
+<script>
+import * as d3 from "d3";
+
+const hour_ms = 60 * 60 * 1000;
+const day_ms = 24 * hour_ms;
+
+function crash_history_to_range(fm_data, start, step, stop, clip) {
+  /*
+   * FuzzManager returns an array of `{ begin: "isodate", count: n }`
+   * objects, where begin is the start of a one hour monitoring
+   * period. Periods where no count was recorded will not be included.
+   *
+   * This function generates a continuous series where gaps are filled
+   * with 0 counts, and values are clipped to a maximum.
+   *
+   * fm_data: Array[ Object { begin: String (ISO date), count: Number } ]
+   * start: Number (timestamp in ms)
+   * step: Number (ms)
+   * end: Number (timestamp in ms)
+   * clip: Number
+   *
+   * returns: Array[ Object { begin: Number (timestamp in ms), count: Number } ]
+   */
+  const result = [];
+  let cp = start,
```

Comment:
```suggestion
  const cp = start,
```

Reply:
`+=` would fail on line 35 if it were `const`.